ipwb/replay.py

```diff
@@ -575,12 +575,14 @@ def showAdmin():
                'htmlCount': htmlCount,
                'earliest': oldest_datetime,
                'latest': newest_datetime}
+
+    # TODO: determine why summary.htmlCount work propagate to template
```

Comment:
Are you suggesting that other attributes of the `summary` object are available except a few?

Reply:
The effect is no longer occuring after removal, perhaps a caching issue.